cardano-cli/src/Cardano/CLI/Shelley/Run/Transaction.hs

```diff
@@ -705,12 +729,15 @@ runTxCalculateMinValue
   -> ExceptT ShelleyTxCmdError IO ()
 runTxCalculateMinValue protocolParamsSourceSpec value = do
   pp <- case protocolParamsSourceSpec of
-    ParamsFromGenesis (GenesisFile f) ->
-      fromShelleyPParams . sgProtocolParams <$>
-        firstExceptT ShelleyTxCmdGenesisCmdError (readShelleyGenesis f identity)
-    ParamsFromFile f -> readProtocolParameters f
+    ParamsFromGenesis (GenesisFile _f) ->
+      panic "Should we make the user specify the era or get the era via reading a txbody"
+     -- fromShelleyPParams . sgProtocolParams <$>
```

Comment:
Should we be calculating the min deposit required for multi-assets via the tx body? We need the era here due to the introduction of the era parameter in `ProtocolParameters era`. We can get it by either 1) specifying a txbody instead of a `Value` 2) Making the user specify the era